### PR TITLE
Issue # 26866: Add documents tab to Vendor master

### PR DESCRIPTION
**Pull request by andersdd**
Linked via CRM account in the same manner as Customer.

### CODE REVIEW

File: guiclient/vendor.cpp [hunk 1]
```diff
@@ -1000,6 +1001,7 @@ bool vendor::sPopulate()
     sFillTaxregList();
 
     _comments->setId(_crmacctid);
+    _documents->setId(_crmacctid);
```
Comment (davidbeauchamp) [inline guiclient/vendor.cpp]: Setting the id isn't enough for _documents, you also have to [setType](http://xtuple.github.io/desktop-programmer-ref/4.9/class_documents.html#a261dc7c55490b602ddab4344abf548fc) for the source document type, which IIRC is "V" for vendors.

Comment (davidbeauchamp) [inline guiclient/vendor.cpp]: But now that I realize you're linking to the crmacctid you should use "CRMA"

Comment (gilmoskowitz) [inline guiclient/vendor.cpp]: That's already done at line 1125 in the .ui.

Comment (davidbeauchamp) [inline guiclient/vendor.cpp]: Ahh you're right about the type, I missed that. Completely agree on using the vendor specific type here.

**Pull request merged**